package.json

```diff
@@ -7,6 +7,7 @@
 		"build:website": "nx build playground-website",
 		"deploy": "gh-pages -d dist/docs -t true",
 		"dev": "nx dev playground-website",
+		"dev:editor": "nx dev interactive-block-playground",
```

Comment:
There will be a regular editor later on and I'd like to avoid renaming it then. I know `dev:interactivity` is mouthful, how about `dev:i`?

Reply:
people can alias their shell commands if they don't like typing full commands. `dev:interactivity` at least is descriptive, whereas `i` is ambiguous and probably conflates with other `i` words.

Reply:
`dev:interactivity` is better IMO 👍 I'll change that!

Reply:
Maybe we'll find a better name for the whole project/feature but should be okay for now.